endpoints/info/bidders_test.go

```diff
@@ -128,9 +143,83 @@ func TestGetBidderAccuracy(t *testing.T) {
 	}
 }
 
+func TestGetBidderAccuracyNoAliases(t *testing.T) {
+	testGetBidderAccuracy(t, "")
+}
+
+func TestGetBidderAccuracyAliases(t *testing.T) {
+	testGetBidderAccuracy(t, "aliasedBidder")
+}
+
+// TestGetBidderAccuracyAlias validates the output for an alias of a known file.
+func testGetBidderAccuracy(t *testing.T, alias string) {
+	bidderInfos := adapters.ParseBidderInfos("../../adapters/adapterstest/bidder-info", []openrtb_ext.BidderName{openrtb_ext.BidderName("someBidder")})
+
+	aliases := map[string]string{}
+	bidder := "someBidder"
+	if len(alias) > 0 {
+		aliases[alias] = bidder
+		bidder = alias
+	}
+
+	endpoint := info.NewBidderDetailsEndpoint(bidderInfos, aliases)
+	req, err := http.NewRequest("GET", "http://prebid-server.com/info/bidders/"+bidder, strings.NewReader(""))
+	if err != nil {
+		t.Fatalf("Failed to create a GET /info/bidders request: %v", err)
+	}
+	params := []httprouter.Param{{
+		Key:   "bidderName",
+		Value: bidder,
+	}}
+
+	r := httptest.NewRecorder()
+	endpoint(r, req, params)
+
+	var fileData adapters.BidderInfo
+	if err := json.Unmarshal(r.Body.Bytes(), &fileData); err != nil {
```

Comment:
Odd it showing this all as additions, where all this test code was actually untouched. We can look into converting it to testify though.

Reply:
ahh ok. If that's the case don't worry about it then. There's plenty of test code we _could_ convert, if we wanted to :).

I'd only be concerned if the problem were growing.

Reply:
Oh, don't worry about converting old code. I didn't realize.

It's showing as additions because... well, they're additions :). Maybe you thought you renamed `TestGetBidderAccuracy` to `testGetBidderAccuracy`? Both exist, though... so it looks like you copy/pasted it rather than renaming.

Reply:
Well, I renamed the function, and then introduced the original name as a new function. Because I needed to write a second test function that duplicated 95% of the code, I converted the original test into a more flexible helper function, called by two calling functions whose names spell out the details of what is being tested in each case. I woudn't think that git would be parsing function names, just noticing that this block of code was untouched but a bunch of lines added inbetween the function name and the block of code.